### PR TITLE
Version 1.0.0-beta: Created connection to database and 2 endpoints

### PR DESCRIPTION
**Pull request by Sz4rv4s**
Version 1.0.0-beta

/api/getAllItems -> returns the items in storage
/api/getAllWarehouses -> returns owned warehouses

### CODE REVIEW

File: partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Alkatresz.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package hu.kosztaauto.partshop.database;
+
```
Comment (marci36912) [inline partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Alkatresz.java:2]: delete one new line, we dont need two

File: partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Alkatresz.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+@Entity
+@Table(name = "alkatresz")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Alkatresz {
+
```
Comment (marci36912) [inline partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Alkatresz.java:16]: same here

File: partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Raktar.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+@Entity
+@Table(name = "raktar")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Raktar {
+
```
Comment (marci36912) [inline partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Raktar.java:17]: if these new lines are intentional then dont mind my comments

File: partshop-backend/src/main/java/hu/kosztaauto/partshop/service/RaktarService.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.util.List;
+
+@Service
+public class RaktarService {
+
+    @Autowired
+    private RaktarRepository raktarRepository;
+
+    public List<Raktar> getAllWarehouses() {
```
Comment (marci36912) [inline partshop-backend/src/main/java/hu/kosztaauto/partshop/service/RaktarService.java:16]: shouldnt public methods be written in pascal case?

**Review by marci36912 — APPROVED** [review]
Approved with suggestions.

File: partshop-backend/src/main/java/hu/kosztaauto/partshop/service/RaktarService.java (hunk 4 above)
Comment (Sz4rv4s) [reply partshop-backend/src/main/java/hu/kosztaauto/partshop/service/RaktarService.java:16]: I prefer camelCase for methods, no matter what scope they have.

File: partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Alkatresz.java (hunk 1 above)
Comment (Sz4rv4s) [reply partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Alkatresz.java:2]: Will work on multiple empty lines, prettyer is enabled to run on start/save so it's shouldn't occur.

**Review by DoneTurnip — APPROVED** [review]

**Pull request merged**